frame/assets/src/mocks.rs

```diff
@@ -12,13 +15,29 @@ use sp_runtime::{
 };
 use system::EnsureRoot;
 
+//
+
+//
+
 type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
 pub type Block = frame_system::mocking::MockBlock<Test>;
 pub type AccountId = u64;
 pub type AssetId = u64;
 pub type Amount = i128;
 pub type Balance = u64;
 
+pub const ALICE: AccountId = 1;
+pub const BOB: AccountId = 2;
+
+pub const EVE: AccountId = 3;
```

Comment:
usually this is C(Charlie, ABC, ABCDE - is 5)

Reply:
could you clarify this?

Reply:
Clarify this please

Reply:
I mean convention is used Charlie (C) before using Eve(E)

Reply:
ok